packages/vite/src/node/ssr/ssrModuleLoader.ts

```diff
@@ -21,7 +21,7 @@ import {
 import { transformRequest } from '../server/transformRequest'
 
 interface SSRContext {
-  global: NodeJS.Global
+  global: typeof globalThis
```

Comment:
This seems not to work :slightly_frowning_face:  
@patak-js Do you know the new equivalent? I didn't find changelogs or something like that :confused:

Reply:
https://github.com/facebook/jest/issues/11640